src/scribe_data/wikidata/query_data.py

```diff
@@ -52,33 +52,21 @@ def execute_formatting_script(formatting_file_path, output_dir):
         The results of the formatting script saved in the given output directory.
     """
     # Determine the root directory of the project.
-    project_root = Path(__file__).parent.parent
+    project_root = Path(__file__).parent.parent.parent
 
-    if sys.platform.startswith("win"):
-        python_executable = sys.executable
-        pythonpath = str(project_root)
+    # Use sys.executable to get the Python executable path.
```

Comment:
Quick nitpick, @axif0: For comments that are their own line can you end them with a period as I've done here? For inline we don't need capitalization of the first word or punctuation at the end, but for Python own line comments are technically sentences :)